ANTLR/src/Compiler/CodeGeneration/JavaScriptCodeGenerationVisitor.java

```diff
@@ -4,19 +4,25 @@
 import Compiler.AntlrGenerated.BuffParser.*;
 import Compiler.ContextualAnalysis.Lambda;
 
-import java.util.ArrayList;
-import java.util.List;
-
-// WARNING: This might be a bad idea !!!
-// This imports all of our enums from LangLexer,
-// so that we don't have to write LangLexer.NUMBERTYPE,
-// instead we can just write NUMBERTYPE
-import static Compiler.AntlrGenerated.BuffLexer.*;
+import static Compiler.AntlrGenerated.BuffLexer.DIVIDE;
+import static Compiler.AntlrGenerated.BuffLexer.LOGAND;
+import static Compiler.AntlrGenerated.BuffLexer.LOGEQ;
+import static Compiler.AntlrGenerated.BuffLexer.LOGGREATER;
+import static Compiler.AntlrGenerated.BuffLexer.LOGGREATEROREQ;
+import static Compiler.AntlrGenerated.BuffLexer.LOGLESS;
+import static Compiler.AntlrGenerated.BuffLexer.LOGLESSOREQ;
+import static Compiler.AntlrGenerated.BuffLexer.LOGNOTEQ;
+import static Compiler.AntlrGenerated.BuffLexer.LOGOR;
+import static Compiler.AntlrGenerated.BuffLexer.MINUS;
+import static Compiler.AntlrGenerated.BuffLexer.MULTIPLY;
+import static Compiler.AntlrGenerated.BuffLexer.PLUS;
+import static Compiler.AntlrGenerated.BuffLexer.POW;
```

Comment:
Er der en mening med single imports i stedet for *? hahahah

Reply:
Jep. Det her er bedre, da det andet inkluderer en masse, som vi ikke bruger.

Reply:
Static imports er farlige 😅 Så det er godt at vi er meget specifikke.